blueprints/theme-a11y-test/blueprint.json

```diff
@@ -13,6 +13,9 @@
 	"features": {
 		"networking": true
 	},
+	"plugins": [
+		"create-block-theme"
+	]
```

Comment:
Hi @zaerl, it looks like a comma is missing from the end of the array literal, and this makes the JSON invalid.

Reply:
Fixed, thanks.